wsutil/reader.go

```diff
@@ -42,7 +46,12 @@ type Reader struct {
 	// header RSV segment.
 	Extensions []RecvExtension
 
-	// TODO(gobwas): add max frame size limit here.
+	// MaxFrameSizeBytes controls the maximum frame size in bytes
+	// that can be read. A message exceeding that size will return
+	// a ErrReadLimit to the application.
+	//
+	// Not setting this field means there is no limit.
+	MaxFrameSizeBytes int64
```

Comment:
Can we drop the `Bytes` suffix? I think it's implies bytes without any other explicit specification?

Reply:
I don't think it hurts to be explicit. But I'm okay if you prefer to drop `Bytes` as long as we mention that in the godoc.

Reply:
Yeah, lets drop it. 👍🏻